src/reportseff/db_inquirer.py

```diff
@@ -215,6 +224,8 @@ def set_sacct_args(self, jobs: list[str]) -> list[str]:
             args += [f"--starttime={self.since}"]
         if self.partition:
             args += [f"--partition={self.partition}"]
+        if self.cluster:
```

Comment:
Please write some db_inquirer tests to exercise the changes here and in partition limits.